README.rst

```diff
@@ -3,216 +3,62 @@ Karapace
-
-* Protobuf schemas are unsupported
+.. _Issue 67: https://github.com/aiven/karapace/issues/67
 
 Setup
 =====
 
-Karapace is a Python project, and requires Kafka for its backend storage. There is also a `Docker setup for development`_.
-
-Requirements
+Using Docker
 ------------
 
-Karapace requires Python 3.6 or later and some additional components in
-order to operate:
-
-* aiohttp_ for serving schemas over HTTP in an asynchronous fashion
-* avro-python3_ for Avro serialization
-* kafka-python_ to read, write and coordinate Karapace's persistence in Kafka
-* raven-python_ (optional) to report exceptions to sentry
-* aiokafka_ for some components of the rest proxy
-
-.. _`aiohttp`: https://github.com/aio-libs/aiohttp
-.. _`aiokafka`: https://github.com/aio-libs/aiokafka
-.. _`avro-python3`: https://github.com/apache/avro
-.. _`kafka-python`: https://github.com/dpkp/kafka-python
-.. _`raven-python`: https://github.com/getsentry/raven-python
-
-Developing and testing Karapace also requires the following utilities:
-requests_, flake8_, pylint_ and pytest_.
-
-.. _`flake8`: https://flake8.readthedocs.io/
-.. _`requests`: http://www.python-requests.org/en/latest/
-.. _`pylint`: https://www.pylint.org/
-.. _`pytest`: http://pytest.org/
-
-Karapace has been developed and tested on modern Linux x86-64 systems, but
-should work on other platforms that provide the required modules.
-
+To get you up and running with the latest release of Karapace, a docker setup is available::
 
-Building
---------
+    docker-compose -f ./container/docker-compose.yml up -d
 
-To build an installation package for your distribution, go to the root
-directory of a Karapace Git checkout and run::
-
-  python3 setup.py bdist_egg
-
-This will produce an egg file into a dist directory within the same folder.
-
-Installation
-------------
-
-Python/Other::
-
-  easy_install dist/karapace-0.1.0-py3.6.egg
-
-On Linux systems it is recommended to simply run ``karapace`` under
-``systemd``::
-
-  systemctl enable karapace.service
-
-and eventually after the setup section, you can just run::
+Then you should be able to reach two sets of endpoints:
 
-  systemctl start karapace.service
+* Karapace schema registry on http://localhost:8081
+* Karapace REST on http://localhost:8082
 
 Configuration
--------------
-
-After this you need to create a suitable JSON configuration file for your
-installation.  Keys to take special care are the ones needed to configure
-Kafka and advertised_hostname.
-
-Each configuration key can be overridden with an environment variable prefixed with
-``KARAPACE_``, exception being configuration keys that actually start with the ``karapace`` string.
-For example, to override the ``bootstrap_uri`` config value, one would use the environment variable
-``KARAPACE_BOOTSTRAP_URI``
-
-
-To see descriptions of configuration keys see section ``config``.  Here's an
-example configuration file to give you an idea what you need to change::
-
-  {
```

Comment:
I removed this sample configuration and just added a link to the sample configuration in the repo. It's easier maintain a single example up-to-date.